src/pages/workspace/reportFields/ReportFieldsAddListValuePage.tsx

```diff
@@ -37,10 +38,18 @@ function ReportFieldsAddListValuePage({
     const {inputCallbackRef} = useAutoFocusInput();
     const [formDraft] = useOnyx(ONYXKEYS.FORMS.WORKSPACE_REPORT_FIELDS_FORM_DRAFT);
 
+    let listValues: string[];
+    if (reportFieldID) {
+        const reportFieldKey = ReportUtils.getReportFieldKey(reportFieldID);
+        listValues = Object.values(policy?.fieldList?.[reportFieldKey]?.values ?? {});
+    } else {
+        listValues = formDraft?.[INPUT_IDS.LIST_VALUES] ?? [];
+    }
```

Comment:
That seems to be a good idea. I tried using `useMemo`, and it seems to avoid unwanted recalculation of `listValues`. I will go on with using useMemo to calculate `listValues`

Reply:
We can put them in `useCallback` and call the function to get the value, but I assumed using `useMemo` makes the code more readable and clean. 

We also use the same pattern here. 
https://github.com/Expensify/App/blob/90fc8fba006b59204ead15efb928f767ba95fc0f/src/pages/workspace/reportFields/ReportFieldsListValuesPage.tsx#L74-L89